### PR TITLE
crw-1995: libsecret-devel required for keytar build on ppc64le and s390x

### PR DESCRIPTION
**Pull request by kcrane**
Signed-off-by: Kirk Crane <kcrane@redhat.com>

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### Happy Path Channel

HAPPY_PATH_CHANNEL=stable

### CODE REVIEW

File: dockerfiles/theia/docker/ubi8/runtime-install-dependencies.dockerfile [hunk 1]
```diff
@@ -12,7 +12,13 @@ ARG SSHPASS_VERSION="1.06"
...
-    tar -xvf sshpass.tar.gz && cd sshpass-"${SSHPASS_VERSION}" && ./configure && make install && cd .. && rm -rf sshpass-"${SSHPASS_VERSION}" && \
-    yum -y clean all && rm -rf /var/cache/yum
+# Install libsecret-devel on s390x and ppc64le for keytar build (binary included in npm package for x86)
+RUN yum install -y sudo git bzip2 which bash curl openssh less \
+    && { [ $(uname -m) == "s390x" ] && yum install -y \
+          https://rpmfind.net/linux/fedora-secondary/releases/34/Everything/s390x/os/Packages/l/libsecret-0.20.4-2.fc34.s390x.rpm \
+          https://rpmfind.net/linux/fedora-secondary/development/rawhide/Everything/s390x/os/Packages/l/libsecret-devel-0.20.4-2.fc34.s390x.rpm || true; } \
+    && { [ $(uname -m) == "ppc64le" ] && yum install -y libsecret https://rpmfind.net/linux/centos/8-stream/BaseOS/ppc64le/os/Packages/libsecret-devel-0.18.6-1.el8.ppc64le.rpm || true; } \
+    && { [ $(uname -m) == "x86_64" ] && yum install libsecret || true; } \
+    && curl -ssl https://netcologne.dl.sourceforge.net/project/sshpass/sshpass/"${SSHPASS_VERSION}"/sshpass-"${SSHPASS_VERSION}".tar.gz -o sshpass.tar.gz \
```
Comment (benoitf) [inline dockerfiles/theia/docker/ubi8/runtime-install-dependencies.dockerfile:22]: Should we avoid specific sourceforge server and use instead primary/aliases servers ?

**Review by benoitf — APPROVED** [review]

**Comment by dmytro-ndp** [comment]
[crw-ce-test --rebuild]

**Comment by nickboldt** [comment]
I've said this before but why are changes to the ubi files causing the alpine tests to fail?

**Comment by nickboldt** [comment]
failed a couple times in local tests, but better now with my fixes... running now, will post console log when it finishes or crashes

**Review by benoitf — APPROVED** [review]

**Comment by nickboldt** [comment]
ran out of disk but got way farther than last time. Log for reference: https://paste.centos.org/view/85e63607

**Pull request merged**